README.md

```diff
@@ -39,56 +39,62 @@ Limber should be accessible via [http://localhost:3001](http://localhost:3001).
 
 ## Initial Setup (using native installation)
 
-Steps must be followed in either the Sequencescape repository or this Limber
-repository, as indicated:
+Because Limber relies on SequenceScape as a backend, SequenceScape and it's jobs can be started first. SequenceScape **will** work standalone without Limber.
 
-1. In Limber, ensure the appropriate version of Ruby is installed. The command
-   here is for `rbenv` but you may want to use a different Ruby version manager:
+> Note that this will require SequenceScape to have already been [setup](https://github.com/sanger/sequencescape/blob/develop/README.md) before
+
+1. The post deploy task will generate required records for Record Loader if they haven't been already. In a Sequencescape terminal, perform the post deploy actions:
 
    ```shell
-   rbenv install
+   bundle exec rake application:post_deploy
    ```
 
-1. In Limber, make the Bundler gem install the dependencies for this project:
+1. In the same terminal, start the local server (will start on port 3000):
 
    ```shell
-   bundle install
+   bundle exec rails s
    ```
 
-1. In Limber, install the yarn dependencies:
+1. Open a **second** SequenceScape terminal, and configure all data required for Limber:
 
    ```shell
-   yarn install
+   bundle exec rake limber:setup
```

Comment:
See` lib/tasks/application.rake`
Line 13
`task post_deploy: %w[record_loader:all limber:setup]`

limber:setup is now done as part of post_deploy. It used be separate.
So you can remove this line.